### PR TITLE
changed backend directory as python module

### PR DESCRIPTION
**Pull request by FadyCoding**
## Describe your changes
Created a python module to launch DebiAI in a standalone state. The previous backend directory has been changed into debiaiServer python module. These changes allow users to start the debiai interface, in this context called debiai-gui, more quickly with a single command.
## Screenshots (if appropriate):
image

## Issue ticket number and link
https://github.com/debiai/DebiAI/issues/205
## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I have increased the version number in the `frontend/package.json` and `backend/swagger.yaml` files
- [x] I have checked that Black, Flake8, Prettier and Cspell are passing

### CODE REVIEW

**Review by Tomansion — CHANGES_REQUESTED** [review]
Excellent job :rocket:

**Comment by Tomansion** [comment]
Complete the pull request description

File: debiaiServer/modules/algoProviders/integratedAlgoProvider/algorithms/regressionErrorMetric.py
Comment (Tomansion) [inline debiaiServer/modules/algoProviders/integratedAlgoProvider/algorithms/regressionErrorMetric.py:1]: :+1:

File: build_and_run.sh
[the remark this answers is not in the document]
Comment (FadyCoding) [reply build_and_run.sh:1]: yes it won't be merged, thank you for noticing it !

File: README.md
[the remark this answers is not in the document]
Comment (FadyCoding) [reply README.md:1]: One `README`  will remain for DebiAI and one will be added for debiai-gui, thank you for your comment ?

File: build_and_run.sh
Comment (Tomansion) [inline build_and_run.sh:1]: The file is here again @FadyCoding  ( ͡ᵔ ͜ʖ ͡ᵔ)''

File: run_debiai_server_dev.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# This file is used to run the debiai-gui server in development mode
+
+from debiaiServer import websrv
+
+websrv.start_server(port=3000, reloader=False, is_dev=True)
```
Comment (Tomansion) [inline run_debiai_server_dev.py:5]: reloader=True

**Review by Tomansion — CHANGES_REQUESTED** [review]

**Review by Tomansion — APPROVED** [review]
ᕦ(ò_ó )ᕤ

**Pull request merged**